docker-entrypoint.sh

```diff
@@ -80,7 +80,16 @@ if expr "$*" : '^java .*/start\.jar.*$' >/dev/null ; then
 		set -- $(cat /jetty-start)
 	else
 		# Do a jetty dry run to set the final command
-		set -- $("$@" --dry-run | sed 's/\\$//' )
+		"$@" --dry-run > /jetty-start
+		EXIT=$?
```

Comment:
I would probably do this:

```shell
"$@" --dry-run > /jetty-start || EXIT=$?
```

That way a non-zero exit code won't cause that line to abort and will capture the exit code. In the case of a zero exit code, `$EXIT` will remain uninitialized, which I think is fine.